src/9-v04x-migration/index.md

```diff
@@ -0,0 +1,92 @@
+---
+date: 2016-02-12
+id: 9-v04x-migration
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "9-v04x-migration", link: "./" }
+tags: gateway, bootstrap, infrastructure
+title: Migrating ipfs.io from go-ipfs 0.3.x to 0.4.0
+author: Lars Gierth
+collection: posts
+---
+
+Good news everyone! We are about to release go-ipfs 0.4.0,
+which contains lots of great changes and enhancements.
```

Comment:
i think we want a separate post about "the great changes and enhancements". can anyone pick this up? maybe @RichardLitt with help of @whyrusleeping and @diasdavid?